pkg/chunk/series_store.go

```diff
@@ -36,7 +36,7 @@ var (
 		Namespace: "cortex",
 		Name:      "chunk_store_index_lookups_per_query",
 		Help:      "Distribution of #index lookups per query.",
-		Buckets:   prometheus.DefBuckets,
+		Buckets:   prometheus.ExponentialBuckets(1, 2, 5),
```

Comment:
Note to myself. Default buckets are:
`.005, .01, .025, .05, .1, .25, .5, 1, 2.5, 5, 10`